redis-wrapper/src/main/java/redis/RedisSingle.java

```diff
@@ -208,6 +208,17 @@ public List<String> getItems(List<String> keys) throws Exception {
         }
     }
 
+    @Override
+    public HashSet<String> getMembers(String key) throws Exception {
+        Set<String> flags;
```

Comment:
Same as the previous comment
```suggestion
        Set<String> items;
```

Reply:
Updated.